cfht2caom2/composable.py

```diff
@@ -144,9 +142,7 @@ def _run_by_builder():
     )
     source = None
     if config.use_local_files:
-        source = data_source.CFHTUseLocalFilesDataSource(
-            config, clients.data_client
-        )
+        source = dsc.UseLocalFilesDataSource(config, clients.data_client)
```

Comment:
Just a style comment (that should have probably been made in a different PR): it is unusual to have verbs in the name of a class.